node/full_node_integration_test.go

```diff
@@ -169,68 +169,110 @@ func TestLazyAggregator(t *testing.T) {
 	}, key, signingKey, proxy.NewLocalClientCreator(app), &cmtypes.GenesisDoc{ChainID: "test", Validators: genesisValidators}, log.TestingLogger())
 	assert.False(node.IsRunning())
 	assert.NoError(err)
-	err = node.Start()
-	assert.NoError(err)
+
+	assert.NoError(node.Start())
 	defer func() {
 		require.NoError(node.Stop())
 	}()
 	assert.True(node.IsRunning())
 
-	require.NoError(err)
-
-	require.NoError(waitForFirstBlock(node.(*FullNode), false))
+	require.NoError(waitForFirstBlock(node.(*FullNode), Header))
 
 	client := node.GetClient()
 
 	_, err = client.BroadcastTxCommit(context.Background(), []byte{0, 0, 0, 1})
 	assert.NoError(err)
-	require.NoError(waitForAtLeastNBlocks(node, 2, false))
+	require.NoError(waitForAtLeastNBlocks(node, 2, Header))
 
 	_, err = client.BroadcastTxCommit(context.Background(), []byte{0, 0, 0, 2})
 	assert.NoError(err)
-	require.NoError(waitForAtLeastNBlocks(node, 3, false))
+	require.NoError(waitForAtLeastNBlocks(node, 3, Header))
 
 	_, err = client.BroadcastTxCommit(context.Background(), []byte{0, 0, 0, 3})
 	assert.NoError(err)
 
-	require.NoError(waitForAtLeastNBlocks(node, 4, false))
+	require.NoError(waitForAtLeastNBlocks(node, 4, Header))
+}
+
+// TestSingleAggregatorTwoFullNodesBlockSyncSpeed tests the scenario where the chain's block time is much faster than the DA's block time. In this case, the full nodes should be able to use block sync to sync blocks much faster than syncing from the DA layer, and the test should conclude within block time
+func TestSingleAggregatorTwoFullNodesBlockSyncSpeed(t *testing.T) {
+	require := require.New(t)
+	aggCtx, aggCancel := context.WithCancel(context.Background())
+	defer aggCancel()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	clientNodes := 2
```

Comment:
You define this and only reference it once by adding 1
```suggestion
	clientNodes := 3
```

Reply:
Updated it everywhere in the file to be consistent.